api/src/database/helpers/geometry.ts

```diff
@@ -87,14 +87,15 @@ class KnexSpatial_SQLite extends KnexSpatial {
 		return this.knex.raw('asgeojson(??.??) as ??', [table, column, column]);
 	}
 }
+
 class KnexSpatial_PG extends KnexSpatial {
 	async supported() {
 		const res = await this.knex.select('oid').from('pg_proc').where({ proname: 'postgis_version' });
 		return res.length > 0;
 	}
 	createColumn(table: Knex.CreateTableBuilder, field: RawField | Field) {
 		const type = field.type.split('.')[1] ?? 'geometry';
-		return table.specificType(field.field, `geometry(${type})`);
+		return table.specificType(field.field, `geometry(${type}, 4326)`);
```

Comment:
Hey @Oreilles 
This PR is supposed to fix on MySQL but you are modifying the Postgres side.
Is this supposed to be like this?

Reply:
Yes, I forgot to mention this in the PR. PostGIS supports explicictly naming the SRID in the type definition, guarantying that no other projection can be inserted - Since we will only ever send WGS84 data ( as per the GeoJSON spec ) it doesn't change anything to our flow, just a safety for use outside Directus!

Reply:
Thank for the clear explanation 🙏